### PR TITLE
chore: automate publishing of packages when merging to main

### PR DESCRIPTION
**Pull request by bhongy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "lint": "turbo run lint",
     "prepare": "husky install",
-    "release": "turbo run build --filter=@formspree/cra-demo^... && changeset publish",
+    "prerelease": "turbo run build --filter=\"./packages/*\"",
```
Comment (bhongy) [inline package.json:15]: https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      - name: Setup Node.js 16.x
-        uses: actions/setup-node@v2
+      - name: Setup Node.js 18.x
+        uses: actions/setup-node@v3
         with:
-          node-version: 16.x
```
Comment (bhongy) [inline .github/workflows/release.yml:20]: 16 is in maintenance. Let's use the current LTS. https://github.com/nodejs/release#release-schedule

File: .changeset/config.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
```
Comment (bhongy) [inline .changeset/config.json:2]: minor: use the version matching what we installed https://github.com/formspree/formspree-js/blob/3d0fe024702e86bede662b3997a8ad33630f7fe4/yarn.lock#L1220

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -1,26 +1,33 @@
...
+      - main
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   release:
     name: Release
+
+    # IMPORTANT: prevent this action from running on forks
+    if: ${{ github.repository_owner == 'formspree' }}
```
Comment (bhongy) [inline .github/workflows/release.yml:15]: https://docs.github.com/en/actions/learn-github-actions/contexts

**Review by colevscode — APPROVED** [review]

**Pull request merged**